### PR TITLE
fixed stream spit | tested on home-server, works fine.

### PR DESCRIPTION
**Pull request by AshtonHarding**
I  am pretty sure it works now. I tested it on my home server.. (Not really sure why, but passthru and fopen just spit out stream data) Try to double check though. Here's an SS of it working on my end. 

http://puu.sh/kgJZR/013f0d11fb.png

### CODE REVIEW

File: b.php [hunk 1]
```diff
@@ -4,17 +4,18 @@
...
+// snags the extension
+$img_extension = pathinfo($name, PATHINFO_EXTENSION);
 
 // send the right headers
 header('Cache-Control: no-cache, no-store, must-revalidate'); // HTTP 1.1
 header('Pragma: no-cache'); // HTTP 1.0
 header('Expires: 0'); // Proxies
-header('Content-Type: ' . $fp['type']);
-header('Content-Length: ' . $fp['bytes']);
+header("Content-type: " . $img_extension);
```
Comment (barrucadu) [inline b.php]: Change that to `"Content-type: image/" . $img_extension` and I think all is good. Makes it work for me (Chrome) at least.

**Comment by AshtonHarding** [comment]
Fixed the text breakout issue. Can be tested here: https://lainchan.org/mod.php?/test/res/948.html

Extremely annoying bug, extremely small fix.

**Comment by AshtonHarding** [comment]
Oops. Didn't mean to close the request.

File: b.php (hunk 1 above)
Comment (AshtonHarding) [inline b.php]: changed that, sorry about the late response, I didn't see this for 3 days.

**Pull request merged**